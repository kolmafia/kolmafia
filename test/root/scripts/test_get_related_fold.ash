$item[january's garbage tote].get_related("fold").dump();
