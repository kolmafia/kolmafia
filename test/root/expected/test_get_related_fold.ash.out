aggregate int [item]
January's Garbage Tote => 1
deceased crimbo tree => 2
broken champagne bottle => 3
tinsel tights => 4
wad of used tape => 5
makeshift garbage shirt => 6
replica January's Garbage Tote => 7
